certbot-dns-cloudflare/setup.py

```diff
@@ -9,7 +9,7 @@
 install_requires = [
     # for now, do not upgrade to cloudflare>=2.20 to avoid deprecation warnings and the breaking
     # changes in version 3.0. see https://github.com/certbot/certbot/issues/9938
-    'cloudflare>=1.5.1, <2.20',
+    'cloudflare<2.20',
```

Comment:
let's maybe still declare a minimum version we're testing/supporting here? whatcha think?

```suggestion
    'cloudflare>=2.19, <2.20',
```

other than that, this lgtm!

Reply:
+1, i previously had it at `==2.19` but this sgtm